ivy/functional/backends/torch/activations.py

```diff
@@ -11,40 +11,32 @@
 # local
 
 
-import ivy
-
-
-def relu(x: torch.Tensor, out: Optional[torch.Tensor] = None) -> torch.Tensor:
-    ret = torch.relu(x)
-    if ivy.exists(out):
-        return ivy.inplace_update(out, ret)
-    return ret
+def relu(x: torch.Tensor) -> torch.Tensor:
+    return torch.nn.functional.relu(x)
 
 
 def leaky_relu(x: torch.Tensor, alpha: Optional[float] = 0.2) -> torch.Tensor:
     return torch.nn.functional.leaky_relu(x, alpha)
 
 
-def gelu(x, approximate: bool = True):
+def gelu(x, approximate: bool = True) -> torch.Tensor:
     if approximate:
         return (
             0.5 * x * (1 + torch.tanh(((2 / np.pi) ** 0.5) * (x + 0.044715 * x**3)))
         )
     return torch.nn.functional.gelu(x)
 
 
-def tanh(x: torch.Tensor, out: Optional[torch.Tensor] = None) -> torch.Tensor:
-    return torch.tanh(x, out=out)
+def tanh(x: torch.Tensor) -> torch.Tensor:
```

Comment:
why are we removing `out` argument?

Reply:
sure I will start fixing it this submodule with latest deep dive

Reply:
I will start working on that after getting your confirmation for Saeed question https://github.com/unifyai/ivy/pull/1461#discussion_r916989309